projects/overlook.md

```diff
@@ -143,76 +166,118 @@ password: overlook2021
+Remember to test all possible outcomes (happy/sad/etc). Ask yourself:
 
-* DOM manipulation / DOM manipulating methods (like `document.querySelector(...)`)
-* Fetch calls
+Does the function return anything?
+Are there different possible outcomes to test for based on different arguments being passed in?
+You are not expected to test:
 
-## Workflow
-You will want to submit PRs to your accountabilibuddy to:
+DOM manipulation / DOM manipulating function (like document.querySelector(...))
+Fetch calls
 
-* You must give your accountabilibuddy collaboration access to your repo.
-* You must submit *at least* 2 PRs to your accountabilibuddy for review.
-* You must wait for your accountabilibuddy to review your PRs, and allow THEM to merge any PRs you submit.
+## Accessibility
 
-It is up to you to decide what changes warrant a PR – remember we want to submit PRs that have significant changes and potential for feedback. Think about what functionality you’re struggling with or have questions about or need help with. As an accountabilibuddy, you are responsible for reviewing at least 2 PRs from your partner.
+Strive for an accessibility audit score of 100% for the dashboard.
 
-**Please also tag your project manager in any PR you make to your buddy.**
+A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad.
 
-## Accessibility
+ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
 
-* Accessibility audits should be at 100% for the dashboard.
-* A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
-* ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
 
+## Minimum Collaboration and Professionalism Expectations
 
-## Due Date
+- Team holds daily standups throughout project.
+- Commits are atomic and frequent, effectively documenting the evolution/progression of the application. There is no more than a 10% disparity in project contributions between teammates.
+- A project board is utilized (and updated throughout the project) with Github issues and labels.
+- Team uses branches, PRs and thorough code reviews to add new code to the main branch.
+- The README is formatted well and at a minimum contains:
+- Overview of project and goals
+- Overview of technologies used, challenges, wins, and any other reflections
+- Screenshots/gifs of your app
+- List of contributors
+- Team collaborates effectively to accomplish the shared goal. Team productively and professionally works through challenges and conflicts to ensure all team members are able to be heard and contribute throughout the project.
+- Instructors are available to offer support and guidance but conversations around what is and what is not working are expected to be led by the team members themselves.
```

Comment:
Maybe we can re-name this just "Minimum Professionalism Expectations" since its a solo project?  
Then we can remove the team-based bullets but maybe keep bullets on commits, project board, and README?

Reply:
Woof - yes. Good catch!